scripts/promote-job.ts

```diff
@@ -72,11 +74,17 @@ export async function createVersionsUpdatePullRequest(
     title,
     versionsChanges,
     branch,
+    qa,
   } = await versionsMutator(currentVersions);
 
-  const body = autoMerge
-    ? `${bodyStart}\n\n// cc: ${releaseOnDuty} — FYI`
-    : `${bodyStart}\n\n${releaseOnDuty} — please approve and merge this PR`;
+  const footers = [];
+  if (qa) {
+    footers.push(`${qaTeam} — please approve this PR for QA`);
+  }
+  if (!autoMerge) {
+    footers.push(`${releaseOnDuty} — please approve and merge this PR`);
+  }
```

Comment:
I still think it's a good idea to `FYI` the release-on-duty team when it's an auto-merge PR, but don't feel strongly about it